tango/integrations/flax/eval.py

```diff
@@ -0,0 +1,144 @@
+from abc import abstractmethod
+from collections import defaultdict
+from itertools import islice
+from typing import Dict, List, Optional, Sequence
+
+import jax
+from flax import jax_utils
+from flax.training.train_state import TrainState
+
+from tango.common.dataset_dict import DatasetDictBase
+from tango.common.exceptions import ConfigurationError
+from tango.common.lazy import Lazy
+from tango.common.registrable import Registrable
+from tango.common.tqdm import Tqdm
+from tango.format import Format, JsonFormat
+from tango.step import Step
+
+from .data import FlaxDataLoader
+from .eval_callback import EvalCallback
+from .util import get_PRNGkey
+
+
+class FlaxEvalWrapper(Registrable):
```

Comment:
Can we have a `FlaxWrapper` that does the things required by both `FlaxTrainWrapper` and `FlaxEvalWrapper`? It's fine to have them separate, but then let's explain the motivation in the docstring.

Reply:
Yeah I think they can be combined together. Will be more clean